src/services/rest-service.ts

```diff
@@ -5,12 +5,17 @@ import { ApiError } from '../Stegside/ApiError';
 const API_URL = '/tiltaksgjennomforing/api';
 const LOGIN_REDIRECT = '/tiltaksgjennomforing/login';
 const HTTP_UNAUTHORIZED = 401;
+const HTTP_CONFLICT = 409;
 
 export default class RestService extends Service {
-    handleAuthorizedResponse(response: Response) {
+    async handleResponse(response: Response) {
         if (response.status === HTTP_UNAUTHORIZED) {
             window.location.href = LOGIN_REDIRECT;
         }
+        if (response.status === HTTP_CONFLICT) {
```

Comment:
Siden `.json()` er async, er det er en mulighet for at `if (!response.ok) { throw ... }` skjerfør den rekker å kaste den første feilmeldingen?

Reply:
`handleResponse` er async, så når `await response.json()` kalles, vil den hoppe ut av metoden, ikke fortsette med `if (!response.ok) { throw ... }`

Reply:
all right!